.eslintrc.js

```diff
@@ -4,16 +4,31 @@ module.exports = {
     node: true
   },
   extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:vue/base',
     'plugin:vue/essential',
+    // 'plugin:vue/recommended',
+    // 'plugin:vue/strongly-recommended',
+    'plugin:vue/vue3-essential',
+    // 'plugin:vue/vue3-recommended',
+    // 'plugin:vue/vue3-strongly-recommended',
+    'plugin:vuetify/base',
+    'plugin:vuetify/recommended',
     '@vue/standard',
     '@vue/typescript'
   ],
   rules: {
     'no-console': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
     'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
     'max-len': ['warn', { code: 120, ignoreRegExpLiterals: true }],
+    // 'vue/attribute-hyphenation': 'off',
     // 'vue/no-deprecated-filter': 'warn',
+    'vue/no-deprecated-slot-scope-attribute': 'warn',
```

Comment:
Without this, we get an eslint error.

With this, we get this warning. I think we should eventually fix this so now it's in our faces 😁 

![image](https://user-images.githubusercontent.com/10002889/233749721-44f05e41-f00a-4b64-a8ed-9a62b7495594.png)